DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -37,7 +37,7 @@ struct NetworkProtectionStatusView: View {
             toggle()
             locationDetails()
 
-            if statusModel.isNetPEnabled && statusModel.shouldShowConnectionDetails && statusModel.ipAddress != nil {
+            if statusModel.isNetPEnabled && statusModel.hasServerInfo && !statusModel.isSnoozing {
```

Comment:
The `ipAddress != nil` check isn't carried over to the `hasServerInfo` implementation. Is it intended?

Reply:
Yep, it was redundant since the server info includes the IP address.